src/main/java/com/amazonaws/kinesisvideo/producer/ClientInfo.java

```diff
@@ -129,4 +134,8 @@ public boolean getLogMetric() {
     public int getAutomaticStreamingFlags() {
         return mAutomaticStreamingFlags.getStreamingFlagValue();
     }
+
+    // public KvsRetryStrategyCallbacks getKvsRetryStrategyCallbacks() {
```

Comment:
This needs to be uncommented if it is being called by the JNI layer [here](https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp/pull/1161/files#diff-9e41d9c63b331f82af0c5aca9ce9d9fe34dedcb3dd2cc965260cc5501ff6a30eR281)

Reply:
Is this not possible?

```
public KvsRetryStrategyCallbacks getKvsRetryStrategyCallbacks() {
      return null
 }
 ```
 
 I think Java null would translate to NULL in JNI/C/C++ layer